### ISSUE TITLE
Back-to-top button does not work on Firefox

### ISSUE THREAD

**Issue by jeffshead** [BookStackApp/BookStack#3988]
### Describe the Bug

The back to the top button does not render in the Firefox browser but it does in Chrome.

### Steps to Reproduce

Visit any long page in BookStack v22.11.1 and scroll down. The button never renders in Firefox.

### Expected Behaviour

The back to the top button should render and work.

### Screenshots or Additional Context

This or a similar bug was [reported/fixed](https://github.com/BookStackApp/BookStack/issues/153) in 2016.

The button does not appear on any page on my site nor on this page of the demo site when using Firefox:
https://demo.bookstackapp.com/books/bookstack-demo-site/page/mixed-content-example-page

I tested with two different Firefox browsers. The button does display in Chrome.

I also tested two other BookStack installs that I found on the Internet.
https://bookstack.surrey.ac.uk/books/rcs-user-guides/page/sftp-using-accesseps-to-transfer-files#bkmrk-sftp
https://docs.punctumbooks.com/books/usage-data/page/monthly-data-collection-and-preparation

The button does display in Firefox on those two Internet sites so I'm guessing those sites must be using BookStack versions that are not affected by this bug.

### Browser Details

Firefox 102.7.0 ESR, Firefox 108.0.1 and 109.0

### Exact BookStack Version

v22.11.1

### PHP Version

Whatever version that comes with the linuxserver/bookstack Docker image.

### Hosting Environment

AlmaLinux 8.7, Docker containers behind Nginx Proxy Manager.

**Comment by ssddanbrown** [comment]
Thanks for reporting @jeffshead.

Tested that provided demo link on FireFox 109.0 (64-bit) on MacOS, works fine as expected.
Will test again on my Ubuntu machine when I'm next on it.

**Comment by ssddanbrown** [comment]
Can confirm it also works fine on Firefox 109.0 (64-bit) on Ubuntu 22.10. 

If you're familiar with browser development tools, it'll be handy to know if any errors are being logged on load/scroll. 
Also worth trying in a private window, or in [troubleshooting mode](https://support.mozilla.org/en-US/kb/diagnose-firefox-issues-using-troubleshoot-mode) to see if this still occurs.

**Comment by jeffshead** [comment]
Dan – so sorry to waste your time. I typically use 2-3 different Firefox installs in addition to other browsers such as Chrome when testing. One of those Firefox installs is supposed to be a vanilla install with zero add-ons. I don't know when, but I mistakenly installed uBlock Origin in the vanilla browser. I did not realize this because the menu bar in that browser didn't have any extra icons. I thought it was all stock.

When I encountered the same issue with all of my Firefox installs and with your demo site, but not on those other two sites that I linked to, I really thought it was a version specific bug in BookStack. Lesson learned – From now on, I will always download a fresh portable version of Firefox or spin up a fresh VM and test again before reporting issues.

FYI - I tracked down the issue to: _**uBlock->Filter lists->Annoyances->Fanboy’s Annoyance**_
